WordPress/UITestsFoundation/Globals.swift

```diff
@@ -6,6 +6,17 @@ import ScreenObject
 // than stored as a reference? 🤔
 public let navBackButton = XCUIApplication().navigationBars.element(boundBy: 0).buttons.element(boundBy: 0)
 
+// Sometimes the Back Button in Navigation Bar is not recognized by XCUITest as an element.
+// This method identifies when it happens and uses a swipe back gesture instead of tapping the button.
+public func navigateBack() {
+    if XCUIApplication().navigationBars.element(boundBy: 0).buttons.count == 1 {
```

Comment:
I had to re-read this code and look at the screenshot to understand why `== 1` was there. At first, I thought it should have been a `.isEmpty` because the comment mentions the button not being there.

How could we make it clearer that we expect there to be 2 buttons? Also, is that a safe assumption to make? 🤔

Reply:
This is the tricky part, making it generic enough since navigation bars will look different in many screens. Back buFor the screenshot in the description, yes, 2 buttons were expected. However there are cases were only one is expected (screenshot below) and for those the `swipe back` will be used, but the `tap` would work as well. 

![image](https://user-images.githubusercontent.com/42008628/145623331-8c7d8257-a880-4dca-a0cf-956e60cc09c7.png)

This was not on purpose but it works like using a safer route (swipe) when the risk is high (there's only 1 button). I thought about keeping only the `swipe back` but for some unknown reason, the second `swipe back` in a row would never work. Documentation is not XCUITests' super power and I gave up on this at some point. 😓